addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5670,6 +5670,21 @@ registry.SnippetMove = SnippetOptionWidget.extend(ColumnLayoutMixin, {
         $overlayArea.prepend($buttons[1]);
         $overlayArea.prepend($buttons[0]);
 
+        // Needed for compatibility (with already dropped snippets).
+        // If the target is a column, check if all the columns are either mobile
+        // ordered or not. If they are not consistent, then we remove the mobile
+        // order classes from all of them, to avoid issues.
+        const parentEl = this.$target[0].parentElement;
+        if (parentEl.classList.contains("row")) {
+            const columnEls = [...parentEl.children];
+            const orderedColumnEls = columnEls.filter(el => this._getItemMobileOrder(el));
+            if (orderedColumnEls.length && orderedColumnEls.length !== columnEls.length) {
+                orderedColumnEls.forEach(el => {
+                    el.className = el.className.replace(/\border(-lg)?-[0-9]+\b/g, "");
```

Comment:
Detail but using `\b` to match classes is not robust. But I see that it's done like that at many places. I've created a task to handle that later ([task-3744675](https://www.odoo.com/web#menu_id=4720&cids=1&action=5050&model=project.task&view_type=form&id=3744675)). We can fix later.